wolfcrypt/src/ecc.c

```diff
@@ -2935,7 +2926,17 @@ int wc_ecc_mulmod_ex(mp_int* k, ecc_point *G, ecc_point *R,
 
    (void)a;
 
-   return sp_ecc_mulmod_256(k, G, R, map, heap);
+#ifndef WOLFSSL_SP_NO_256
+    if (mp_count_bits(modulus) == 256) {
+        return sp_ecc_mulmod_256(k, G, R, map, heap);
+    }
+#endif
+#ifdef WOLFSSL_SP_384
```

Comment:
Don't include the SP_384 code as it is not implemented in 4.1.0.

Reply:
Eliminated it.